contentrepo/settings/base.py

```diff
@@ -32,7 +32,8 @@
     "wagtail_content_import.pickers.local",
     "wagtail.admin",
     "wagtail",
-    "wagtail.locales",
+    "wagtail_localize",
+    "wagtail_localize.locales",
```

Comment:
According to https://wagtail-localize.org/latest/tutorial/2-configure-wagtail-localize/ , this should be placed between `search` and `wagtail.contrib.forms`.

It also says that `wagtail_localize.locales` is temporary and will be removed in future, so maybe we should stick with `wagtail.locales`?